### PR TITLE
lang_update: when import is set, consider missing strings too

### PR DESCRIPTION
**Pull request by flodolo**

### CODE REVIEW

File: app/scripts/lang_update [hunk 1]
```diff
@@ -104,16 +104,35 @@ foreach ($file_list as $current_filename) {
...
-            $common_strings = array_intersect(
-                array_keys($import_data['strings']),
-                array_keys($locale_data['strings'])
+
+        if (count($import_files) > 0) {
+            /*
+                Add to $locale_data strings that are available in the
+                reference file but are missing in the localized one. This
+                way we can import them before building the file.
+                Normally this is not necessary, since LangManager::buildLangFile
```
Comment (TheoChevalier) [inline app/scripts/lang_update]: I assume "normally" is when we do not import anything?

Comment (flodolo) [inline app/scripts/lang_update]: Yes. When you have to import anything, strings are added later when building the .lang file.

**Comment by TheoChevalier** [comment]
Ok, LGTM :) I did a quick import test, this worked smoothly

**Comment by flodolo** [comment]
Thanks.

**Pull request merged**